README.md

```diff
@@ -21,15 +21,16 @@ gem "merch_calendar"
 
 ## Usage
 
-For converting a date into a `MerchWeek` object.
+For converting a date into a `MerchWeek` object.  `MerchWeek` object can take a date can translate to what merch week the date falls into. 
```

Comment:
> object can take a date can translate 

I'm not sure what you were trying to say here, but it looks in need of revising.

Reply:
I ended up deleting that sentence that I added.  I think the ruby text will speak for itself